procedures/unit-testing-basics.ipf

```diff
@@ -931,6 +931,36 @@ Function TEST_CASE_PROTO_MD_WV([wv])
 	abortNow()
 End
 
+Function TEST_CASE_PROTO_MD_WVTEXT([textWave])
```

Comment:
Can we use `wv` as optioal parmeter name as well? This makes writing test cases which use that feature easier as I don't have to remember the specialized name.

Reply:
Changed optional parameters to wv as well.
Though the name in the test case/reentry functions do not matter as it is an optional parameter.